src/main/java/mate/academy/dao/impl/BookDaoImpl.java

```diff
@@ -0,0 +1,135 @@
+package mate.academy.dao.impl;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Book;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class BookDaoImpl implements BookDao {
+    @Override
+    public Book create(Book book) {
+        String createRequest = "INSERT INTO books "
+                + "(title, price) values(?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+             PreparedStatement createManufacturerStatements =
```

Comment:
Since you have only one statement in the method, you can name it just "statement" to improve code readability.
And, you have book create statement, is it actually about manufacturers?